numba/np/arraymath.py

```diff
@@ -4251,3 +4251,31 @@ def impl(a, b):
         return _cross2d_operation(a_, b_)
 
     return impl
+
+
+@overload(np.trim_zeros)
+def np_trim_zeros(a, trim='fb'):
+    if not isinstance(a, (types.Array, types.Sequence, types.Tuple)):
+        raise TypeError('The first argument must be an array-like')
+    if hasattr(a, 'ndim') and a.ndim > 1:
+        raise TypingError('array must be 1D')
+
+    def impl(a, trim='fb'):
+        a_ = np.asarray(a)
+        first = 0
+        if 'f' in trim:
```

Comment:
From [this line](https://github.com/numpy/numpy/blob/v1.18.1/numpy/lib/function_base.py#L1608) it seems the numpy implementation works with upper and lower case letters for `trim`

```
>>> np.trim_zeros(np.array([0, 1, 2, 3, 0, 0]), trim="FB")
array([1, 2, 3])
```